### PR TITLE
Validation: Added Unique Descriptions

### PR DESCRIPTION
**Pull request by JustPies**
Fixed:

- description-missing errors

This PR fixes 30 issues across 30 files and is part of ongoing Validation cleanup efforts. No other content has been edited or added.

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/advanced/typography-how-to-topics.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ f1_keywords:
 helpviewer_keywords: 
   - "typography [WPF], how-to topics"
 ms.assetid: 82d50325-7cb2-4975-aea3-027c00e6bbfc
+description: Learn  how to use Windows Presentation Foundation (WPF) support for rich presentation of text in your applications.
```
Comment (brittmsantos) [inline dotnet-desktop-guide/framework/wpf/advanced/typography-how-to-topics.md:9]: Delete extra space in "Learn  how"

**Comment by JustPies** [comment]
Global effort to fix validation errors

@adegeo

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Comment by jmart1428** [comment]
@adegeo Please let us know if you'd like to see any additional changes in this PR. If no changes are required, you are welcome to merge. Thank you for your feedback!

File: dotnet-desktop-guide/framework/wpf/advanced/use-automatic-layout-overview.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ helpviewer_keywords:
   - "layout [WPF], automatic"
   - "automatic layout [WPF]"
 ms.assetid: 6fed9264-18bb-4d05-8867-1fe356c6f687
+description: Learn the guidelines for developers on how to write WPF application design for the use of automatic layouts.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/advanced/use-automatic-layout-overview.md:8]:
```suggestion
description: Describes the guidelines for developers on how to write WPF application design for the use of automatic layouts.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**